Food_Tracker_App/app/src/main/java/com/example/foodtracker/ui/ingredients/testDialog.java

```diff
@@ -58,7 +58,8 @@ public Dialog onCreateDialog(@Nullable Bundle savedInstanceState) {
                 .setNeutralButton("ok", new DialogInterface.OnClickListener() {
                     @Override
                     public void onClick(DialogInterface dialogInterface, int i) {
-                        Ingredient egg = new Ingredient("egg");
+                        Ingredient egg = new Ingredient("egg", 1.6, "pantry"," food"
+                        ,1,"02/02/2022");
                         listener.onOkPressed(egg);
```

Comment:
this line (listener.OkPressed(egg)) crashes up when "ok" button is pressed - since it's fragment to fragment sending data, would using bundles to send information work?

Reply:
I have made a fix for this to make the ingredients page an activity.